contracts/interfaces/IWalletCustomLogic.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier:MIT
+pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "@rsksmart/rif-relay-contracts/contracts/interfaces/IForwarder.sol";
+
+/**
+ * Interface defining the methods that should be implemented
+ * in order to provide custom logic to a CustomSmartWallet
+ */
+interface IWalletCustomLogic {
```

Comment:
why not use the [rif-relay-contracts/contracts/interfaces/IWalletCustomLogic.sol](https://github.com/rsksmart/rif-relay-contracts/blob/main/contracts/interfaces/IWalletCustomLogic.sol) directly instead of replicating it in this file?

Reply:
👆🏽

Reply:
done